charts/fluent-bit/values.yaml

```diff
@@ -482,3 +481,12 @@ initContainers: []
 #     command: ['kubectl', 'version']
 
 logLevel: info
+
+hotReload:
+  enabled: false
+  image:
+    repository: ghcr.io/jimmidyson/configmap-reload
```

Comment:
We should indicate this in the release notes as significant that users may get another image in their stack if they enable hot-reload.

Do we want to bring this into the fluent organisation as a fork just to fully control?

Reply:
We don't really have release notes (see above). It's used by multiple other CNCF (or related) projects, Grafana & Thanos spring to mind and there is either a Changuard version now or pending. I don't think it scales for everyone to build their own helper images. If it's a big enough concern Fluent Bit could add native support for watching the config.